share/man/man4/ix.4

```diff
@@ -86,13 +87,49 @@ Intel(R) 82598EB 10 Gigabit AF Network Connection
 .It
 Intel(R) 82598EB 10 Gigabit AT CX4 Network Connection
 .El
+.Sh LOADER TUNABLES
+The
+.Nm
+driver supports the following sysctl variables:
+.Bl -tag -width "hw.ix.allow_unsupported_sfp"
```

Comment:
is this line intentional?

Reply:
This one is how felix typed it and the syntax looks perfect to me. What is wrong with it?

Reply:
I guess I don't know what is going on here, why does it reference `hw.ix.allow_unsupported_sfp` which doesn't appear again until further down.

Reply:
The `-width` argument in a `.Bl -tag` (begin tagged list) sets the width of the first longest item in the list so that the second column of rendered list is aligned beautifully.